connectors/es/sink.py

```diff
@@ -525,6 +527,93 @@ async def get_docs_incrementally(self, generator):
 
         await self.queue.put("END_DOCS")
 
+    async def get_access_control_docs(self, generator):
+        """Iterate on a generator of access control documents to fill a queue with bulk operations for the `Sink` to consume.
+
+        A document might be discarded if its timestamp has not changed.
+        """
+        logger.info("Starting access control doc lookups")
+        generator = self._decorate_with_metrics_span(generator)
+
+        self.sync_runs = True
+
+        start = time.time()
```

Comment:
this block sounds like a good candidate to use `@timed_execution` or the span

Reply:
Already using it on the generator, so just removing this part 👍